asreviewcontrib/insights/plot.py

```diff
@@ -186,7 +186,7 @@ def _plot_wss(ax, labels, x_absolute=False, y_absolute=False):
         yticks = [0, 0.2, 0.4, 0.6, 0.8, 1.0]
 
     ax.step(x, y, where='post')
-    ax.set_title("WSS")
+    ax.set_title("Work Saved over Sampling (WSS) versus Recall")
```

Comment:
```suggestion
    ax.set_title("Work Saved over Sampling (WSS) given Recall")
```

What is preferred? Ignore if disagree.